NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
I just want to give hornetq contributor the credit for their contributions on what became Artemis today.

We took a snapshot of hornetq during the Artemis, as it seemed the right decision at the time given the lengthy history logs existing on the donation. I think it's reasonable to include a word about their contributions.